app/views/posts/index.html.erb

```diff
@@ -1,7 +1,31 @@
-<h1>Post List</h1>
+<div class="posts-container">
 
-<% @posts.each do |post|%>
-    <div class="post">
-        <%= post.text %>
+    <div class="user-box">
+        <img class="user-img" src=<%= @user.photo%> alt="user"/>
+        <div class="user-info-box">
+            <h2 class="user-name"><%= @user.name %></h2>
+            <p class="user-post-count">Number of posts: <%= @user.posts_counter%></p>
+
```

Comment:
- [ ] Kindly note that you are calling a @user (instance variable) in a post view, this is giving me an error, saying undefined method photo for nil class. When you take out this piece of code, the app works as expected. Let's make changes an ask for another review.

![post-index](https://github.com/AbdimulhinYussuf3675/Blog-app/assets/53124548/397917cc-7fca-4a28-9683-577e84bf8250)

Reply:
Hello @elisha2kyakpo1, Thanks for the review. Please I don't understand which code am supposed to remove. Can please explain to me further